exe/solr_wrapper

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env ruby
 
+if ENV['RAILS_ENV'] && ENV['RAILS_ENV'] == 'production'
```

Comment:
This gem doesn't require rails, so a dependency on `RAILS_ENV` is inappropriate.